app/templates/components/banner.html

```diff
@@ -18,7 +18,7 @@ <h2 class="alerts-notification-banner__title govuk-heading-m govuk-!-margin-bott
   {% set title %}
     {% for alert_date in dates_of_test_alerts|sort %}
       {% if loop.index == 1 %}
-        On {{ alert_date.as_alert_date_lang }}, there will be a national test of the UK Emergency Alerts service.
```

Comment:
I added this function `as_alert_date_lang` and it is only used for this place in the code and nowhere else.  As such you can just replace the format inside that function (or just delete the function).

Reply:
Unused method removed